examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -107,6 +112,36 @@ class ClusterCommand : public ModelCommand, public chip::app::CommandSender::Cal
         return CHIP_NO_ERROR;
     }
 
+    template <class T>
+    CHIP_ERROR SendGroupCommand(chip::GroupId groupId, chip::FabricIndex fabricIndex, chip::NodeId senderNodeId,
+                                chip::ClusterId clusterId, chip::CommandId commandId, const T & value)
+    {
+        chip::app::CommandPathParams commandPath = { 0 /* endpoint */, groupId, clusterId, commandId,
+                                                     (chip::app::CommandPathFlags::kGroupIdValid) };
+
+        chip::Messaging::ExchangeManager * exchangeManager = chip::app::InteractionModelEngine::GetInstance()->GetExchangeManager();
+
+        ChipLogDetail(chipTool, "Sending command to Group %u, on Fabric %x, for cluster %u with commandId %u", groupId, fabricIndex,
+                      clusterId, commandId);
```

Comment:
Please use ChipLogFormatMEI/ChipLogValueMEI for the cluster and command ids.

Please either use %u for the fabric or prefix with 0x to make it clear it's hex.

Reply:
removed debug log